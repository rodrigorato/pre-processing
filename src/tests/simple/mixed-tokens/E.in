import java.util.*;
import java.util.concurrent.*;

alias display = System.out.println;

public class Example2 {

    public static void main(String[] args) {
		//Another great idea from our beloved client
		var y = new ArrayList<ArrayList<Integer>>();
		foo(1,2,3);
	        int curDate = 0;
	        if (curYear > 2018) {
	            display(curYear);
	        }
    }

    alias i = int;

    static void foo(i a, int b, int c) {
		String str = #"First #{a}, then #{a+b}, finally #{b*c}.";
		display(str);
        display(byebye);
        display((bar));
    }

    alias Cache = ConcurrentSkipListMap<String,List<Map<String,Map<String,Map<String,Object>>>>>;

    public static Cache mergeCaches(Cache a, Cache b) {
		var temp = new Cache();
		return temp;
    }
}