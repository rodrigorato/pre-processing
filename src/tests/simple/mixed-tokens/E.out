import java.util.*;
import java.util.concurrent.*;



public class Example2 {

    public static void main(String[] args) {
		//Another great idea from our beloved client
		ArrayList<ArrayList<Integer>> y = new ArrayList<ArrayList<Integer>>();
		foo(1,2,3);
	        int curDate = 0;
	        if (curYear > 2018) {
	            System.out.println(curYear);
	        }
    }

    

    static void foo(int a, int b, int c) {
		String str = "First " + (a) + ", then " + (a+b) + ", finally " + (b*c) + ".";
		System.out.println(str);
        System.out.println(byebye);
        System.out.println((bar));
    }

    

    public static ConcurrentSkipListMap<String,List<Map<String,Map<String,Map<String,Object>>>>> mergeCaches(ConcurrentSkipListMap<String,List<Map<String,Map<String,Map<String,Object>>>>> a, ConcurrentSkipListMap<String,List<Map<String,Map<String,Map<String,Object>>>>> b) {
		ConcurrentSkipListMap<String,List<Map<String,Map<String,Map<String,Object>>>>> temp = new ConcurrentSkipListMap<String,List<Map<String,Map<String,Map<String,Object>>>>>();
		return temp;
    }
}